docs/docs/concepts/conversation-repair.mdx

```diff
@@ -144,15 +140,34 @@ a flow with the same name as the pattern:
 - `pattern_clarification` for clarification.
 - `pattern_internal_error` for internal errors.
 
+
 The syntax for each of these flows is the same as any other flow.
 
+::info
+
+Conversation repair cases are expected to work out-of-the-box. This means that if the [default behaviour](../concepts/conversation-repair.mdx#reference-default-pattern-configuration) is good enough for the assistant's use case, then the flow corresponding to the pattern handling the repair case is not needed in the assistant's project directory.
+
+::info
+
+
+### Modifying default behaviour
+
+It is possible to override the default behaviour of each conversation repair case by creating a flow with the same name as that of the pattern used to handle the corresponding case, like `pattern_correction`. If the pattern uses a default action which needs to be modified, you can override the implementation of the default action by implementing a new custom action and use that custom action in the flow.
+
 :::info
 
-Since mose of these patterns interrupt another flow, they should be
+Make sure the assistant is re-trained after the modification is completed.
+
+:::
+
+:::info
+
+Since most of these patterns interrupt another flow, they should be
 kept short and simple.
 
 :::
```

Comment:
looks like a mistake?

Reply:
Umm no, I wanted to add two info callouts and move the existing one down. Is that not valid?

Reply:
ah ok, I read the diff wrong